changelogs/fragments/3946-mattermost_attachments.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - mattermost - add the possibility to send attachments instead of text messages. (https://github.com/ansible-collections/community.general/pull/3946)
```

Comment:
```suggestion
  - mattermost - add the possibility to send attachments instead of text messages (https://github.com/ansible-collections/community.general/pull/3946).
```